### DISCUSSION TITLE
生产环境使用beta是否合适

### DISCUSSION THREAD

**Discussion by Nofated095** [vuepress/core#955] [Q&A] [answered]
如题，我打算在Production里面使用vuepress-next会不会那些beta的不稳定组件会导致production不稳定

**Comment by Nofated095** [comment]
并且想问一问大概什么时候会出到正式版

**Comment by yzqdev** [comment, marked as answer]
Vuepress is ready to go to production. And it's better to always use the latest version of vuepress and keep up with the api changes.

**Reply by taozuhong** [reply]
Most of it's plugins have been ported to VuePress-Next and Vue3?
